src/herder/test/HerderTests.cpp

```diff
@@ -5554,7 +5554,7 @@ TEST_CASE("SCP message capture from previous ledger", "[herder]")
     simulation->addConnection(validatorCKey.getPublicKey(),
                               validatorBKey.getPublicKey());
     simulation->crankUntil(
-        [&]() { return C->getLedgerManager().getLastClosedLedgerNum() == 3; },
+        [&]() { return C->getLedgerManager().getLastClosedLedgerNum() >= 3; },
```

Comment:
Why is this required? Was the test flaky before?

Reply:
It's not flaky, but it switched from passing to falling with the protocol bump. I've just double-checked, and we end up closing two ledgers within a single crank iteration. I don't think this is an issue - probably it has been on the edge before. It's interesting that this seems to be deterministic, but I'm not familiar with the whole virtual clock machinery enough to be sure whether we should be surprised by this or not.